### PR TITLE
Fix 2 typos

### PR DESCRIPTION
**Pull request by ASDosjani**

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

**Review by howels — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
